google-beta/resource_project_service_identity_test.go

```diff
@@ -0,0 +1,38 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccProjectServiceIdentity_basic(t *testing.T) {
+	t.Parallel()
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckHealthcareDatasetDestroyProducer(t),
```

Comment:
```suggestion
```
(since there is no destroy for this resource, we should just remove the CheckDestroy rather than calling a different one)

Reply:
Oh whoops, I missed that. I've modified the upstreaming PR to remove it, so the Magician will correct this repo regardless of whether it's made on this PR specifically or not.